src/components/FeatureSection.jsx

```diff
@@ -1,39 +1,92 @@
-import * as React from 'react'
+import * as React from "react";
+import {
+  leakProofIcon,
+  qualityTrustIcon,
+  strengthIcon,
+  uvResistantIcon,
+} from "./common/image";
 
 const FeatureSection = () => {
   return (
     <section className="bg-white dark:bg-gray-900">
       <div className="py-8 px-4 mx-auto max-w-screen-xl sm:py-16 lg:px-6">
-          <div className="max-w-screen-md mx-auto mb-8 text-center lg:mb-16">
-              <h2 className="mb-4 text-4xl tracking-tight font-extrabold text-gray-900 dark:text-white">Designed for business teams like yours</h2>
-              <p className="text-gray-500 sm:text-xl dark:text-gray-400">Here at Flowbite we focus on markets where technology, innovation, and capital can unlock long-term value and drive economic growth.</p>
+        <div className="max-w-screen-md mx-auto mb-8 text-center lg:mb-16">
+          <h2 className="mb-4 text-4xl tracking-tight font-extrabold text-gray-900 dark:text-white">
```

Comment:
Instead of duplicating, create a list of items which will have icon, heading and text and iterate over that list to render everything. It is easier to scale and debug/fix in future

Reply:
it's done